Applications/Spire/Source/Ui/EditableTableView.cpp

```diff
@@ -262,6 +274,32 @@ EditableTableView::EditableTableView(
   get_header().get_item(0)->set_is_resizeable(false);
   get_header().get_widths()->set(0, scale_width(26));
   set_style(*this, TABLE_VIEW_STYLE());
+  setFocusProxy(&get_body());
```

Comment:
Can you explain why this is needed? Why isn't this used by the TableView?

Reply:
The purpose is to have TableBody get keyboard input (Array keys and Home/End) when the EditableTableView has focus. I think it is more appropriate to do it by the TableView.